media_driver/cmake/media_utils.cmake

```diff
@@ -59,5 +60,15 @@ macro (MediaAddCommonTargetDefines target)
     endif()
 endmacro()
 
+# find external libs, if not found, print an error message and abort cmake
+macro (find_external_libs EXTERNAL_LIBS)
+    string (REPLACE " " ";" EXTERNAL_LIBS_LIST "${EXTERNAL_LIBS}")
+    foreach (LIB ${EXTERNAL_LIBS_LIST})
+        pkg_check_modules (${LIB} REQUIRED)
+        if (NOT ${LIB}_FOUND)
+            message (FATAL_ERROR "cannot find enternal lib: ${LIB}")
```

Comment:
Are you sure that this code will be executed at all after failure on REQUIRED? I very much doubt.